src/cli.ts

```diff
@@ -94,10 +96,15 @@ const addReposOption = (command: program.Command) => {
   );
 };
 
+const addUpstreamOwnerOption = (command: program.Command) => {
+  return command.option('--upstreamOwner <upstreamOwner>', 'Upstream Owner can be passed incase of trying to raise PR from fork to upstream');
```

Comment:
Perhaps this option should be called forkOwner instead of upstreamOwner to provide more clarity on its use.  When forkOwner is defined, PR will be generated off of fork to upstream instead of creating a branch on upstream.